SalesforceCommerceCloud/pwa-kit
# W-14337993: Fix a11y for tab focus, implement gift option for basket

# Description
This PR fixes a11y issues in https://gus.lightning.force.com/a07EE00001cqESEYA2


# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- Check out code
- Add item to cart
- Go to Cart page
- Check the "thiis is a gift" checkbox using keyboard
- Tab to the 'info' icon next to promotion applied in Order Summary. Make sure the screen reader can regconize content in the pop-up

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
